kubeflow/applications.tf

```diff
@@ -0,0 +1,243 @@
+
+module "admission_webhook" {
+  source     = "git::https://github.com/canonical/admission-webhook-operator//terraform?ref=track/1.9"
+  model_name = var.create_model ? juju_model.kubeflow[0].name : local.model_name
```

Comment:
`model_name` has been changed to `model` in the spec, as newer version of the provider will slowly change towards using UUIDs. This comment is valid for the rest of the project and the underlying modules.

Reply:
Thanks for catching this. I 'll update this solution module to use `model` and then file an issue to update all charm modules at some point with the proper attribute name.

Reply:
Filed issue https://github.com/canonical/analytics-terraform-solutions/issues/8